test/unit/tile.js

```diff
@@ -128,6 +128,22 @@ describe('Tile', function () {
     });
   });
 
+  it('Valid formats pass', function () {
+    ['jpeg', 'png', 'webp'].forEach(function (format) {
+      assert.doesNotThrow(function () {
+        sharp().toFormat(format).tile();
+      });
+    });
+  });
+
+  it('Invalid formats fail', function () {
+    ['zoinks', 1, 'tiff', 'raw'].forEach(function (format) {
```

Comment:
I think the `zoinks` and `1` values here end up testing the existing `toFormat` function rather than your additions to the `tile` function.